test/services/api/api.service.unit.ts

```diff
@@ -0,0 +1,9 @@
+import { createOAuthApiClient } from "../../../src/services/api/api.service";
+import { getSessionRequest } from "../../mocks/session.mock";
+
+describe ("Test node session handler authorization for private sdk", () => {
```

Comment:
can you add tests to test the error gets thrown? need to cover session.data being undefined, signInInfo being undefined and AccessToken being undefined. We had similar issues with date formatter where we didn't test all paths through the function and that caused the bug.

Reply:
Ok added test now passing.